src/software/backend/radio_backend.h

```diff
@@ -11,6 +11,14 @@ class RadioBackend : public Backend
 
     RadioBackend();
 
+    /**
+     * Creates a new PrimitiveGenerator
+     *
+     * @param controller_input_config The config for the RadioBackend
+     */
+    RadioBackend(
```

Comment:
The RadioBackend shouldn't need the `HandheldControllerInputConfig` so I don't think we need to add a constructor. The `HandheldControllerInputConfig` controls how the `handheld_controller` generates primitives, but by the time they reach the backend they should just be seen as "normal primitives".

Reply:
In `handheld_control_main.cpp`, the original code passes in `controller_config` to `std::make_shared<HandheldControllerBackend>(controller_config)`. Without the constructor, how would we pass in `controller_config` into `RadioBackend`? Or is this already handled by `controller` and `primitive_generator` in the same `handheld_control_main.cpp` file?

Reply:
The `RadioBackend` doesn't need the `controller_config`, only the `HandheldControllerBackend` needed it.